Makefile

```diff
@@ -10,7 +10,7 @@ GZIP ?= gzip -9 -n
 DIFFOSCOPE ?= diffoscope
 STRIP ?= strip
 
-CFLAGS ?= -O3 -g
+CFLAGS ?= -O3
```

Comment:
Maybe this is combination of -O3 and -g. I think we can add a DEBUG=1 option to create a  debug build that can be used locally.

And we probably need to file Apple bug for this, this looks wrong.

Reply:
Added a commit `Makefile: support specifying DEBUG=1`.

Not sure if this is a bug or an expected limitation.